ivy.xml

```diff
@@ -11,7 +11,6 @@
   <dependency org="javax.servlet" name="javax.servlet-api" rev="3.1.0" />
   <dependency org="javax.mail" name="mail" rev="1.4.5" />
   <dependency org="org.dom4j" name="dom4j" rev="${dom4j.version}" />
-  <dependency org="log4j" name="log4j" rev="1.2.16" />
```

Comment:
better to revert this change, as this repo has some jsp and java files so maybe this dependency is needed here

Reply:
Added back log4j dependency